src/main/java/frc/robot/subsystems/Module.java

```diff
@@ -1,5 +1,17 @@
 package frc.robot.subsystems;
```

Comment:
I noticed that all the loggs go Swerve/ID/item. I think it would be better to have it Swerve/item/ID becasue you are more likely to want to look at items of the same type rathe rather than all items of a spcific module